ecs-update/ecs-update_test.go

```diff
@@ -380,3 +380,148 @@ func TestReplaceECSFields(t *testing.T) {
 		t.Errorf("unexpected changes found in %s", filepath.Base(fieldsFile.Path()))
 	}
 }
+
+func TestRemoveECSFields(t *testing.T) {
```

Comment:
Here's an edge case to test/handle.

Given this content:

```
- name: container
  type: group
  fields:
    - name: id
      external: ecs
```

After the `id` is removed, validation of the package will fail with

> field 0.fields: Invalid type. Expected: array, given: null

It looks like the empty parent node needs purged too.